packages/python-pulpcore/python-pulpcore.spec

```
@@ -126,9 +126,15 @@ Obsoletes:      python3-%{pypi_name} < %{version}-%{release}
 %if 0%{?rhel} == 8
 Obsoletes:      python39-%{pypi_name} < %{version}-%{release}
 %endif
+
 # Pulp-file and Pulp-certguard are now part of pulpcore
 Obsoletes:      python3.11-pulp-file < 1.16.0-1
+Provides:       python%{python3_pkgversion}-pulp-file
+Provides:       pulpcore-plugin(file)
```

when it comes to the installer, we only "need" the `pulpcore-plugin(<name>)` notation:
https://github.com/theforeman/puppet-pulpcore/blob/21aa39e28f19b40bf328ebba514465fb9cfb9fe2/manifests/plugin.pp#L15

and I'd love those to be versioned, so
```suggestion
Provides:       python%{python3_pkgversion}-pulp-file = %{version}
Provides:       pulpcore-plugin(file) = %{version}
```

---

based on "pulp-????-client" version for both of these plugins in pypi, agreed.